distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc

```diff
@@ -22,6 +22,11 @@ These steps will demonstrate:
 
 These are the basic requirements to set up the environment to run a ${branding}.
 
+[WARNING]
```

Comment:
❓ ❗️ Will that be inherited by all upstream projects? If so than it would be a problem for those that do not enable the security manager.

Reply:
I think we should consider security manager a best practice and less of a feature that can be turned on and off. I would argue if upstream projects are not using the security manager, that it's on them to fix it, not for DDF to support extensibility points in the scenario that they turn it off.

Reply:
I also agree that using the security manager should be considered the norm not the exception.  I will leave the documentation as-is.

Reply:
For now, that is good enough for me. I just wanted to make sure we were aware of the implications. We can revisit if we ever see DDF use in non-defense type of applications where such tight security is not required.